static/app/components/feedback/list/feedbackListHeader.tsx

```diff
@@ -62,11 +62,12 @@ function HasSelection({checked, mailbox}) {
       <Flex gap={space(1)} justify="flex-end">
         <ErrorBoundary mini>
           <Button
-            onClick={() =>
+            onClick={() => {
               mailbox === 'resolved'
                 ? resolve(GroupStatus.UNRESOLVED)
-                : resolve(GroupStatus.RESOLVED)
-            }
+                : resolve(GroupStatus.RESOLVED);
+              checked.forEach(c => toggleChecked(c));
```

Comment:
Lets make a function inside of `useListItemCheckboxState` like `uncheckAll` (and we'll also have a `checkAll` soon) because we're doing this here, and also inside of feedbackListHeader right now